pkg/operation/botanist/component/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -168,13 +168,16 @@ type ControllerWorkers struct {
 	Endpoint *int
 	// GarbageCollector is the number of workers for the GarbageCollector controller.
 	GarbageCollector *int
-	// Namespace is the number of workers for the Namespace controller.
+	// Namespace is the number of workers for the Namespace controller. Set it to '0' in order to disable the controller
+	// (only works when cluster is workerless).
```

Comment:
We want to disable namespace controller for `gardener-operator`?

Reply:
Not really, I just added this implementation for consistency. For now, we'll only disable the SA-token controller.

Reply:
I didn't get the consistency part, You have done this for only three controllers.

Reply:
For the controllers in `--controllers=namespace,serviceaccount,serviceaccount-token,clusterrole-aggregation,garbagecollector,csrapproving,csrcleaner,csrsigning,bootstrapsigner,tokencleaner,resourcequota`, this PR has made the number of workers configurable for the `Namespace`, `ServiceAccountToken`, and `ResourceQuota` controller. For the others, the number of workers aren't configurable yet (because not needed, maybe in the future).